lxd/api_project.go

```diff
@@ -1018,6 +1018,21 @@ func projectNodeConfigDelete(d *Daemon, name string) error {
 			return fmt.Errorf("Failed to load local node config: %w", err)
 		}
 
+		// Unmount the project-specific storage volumes.
+		if config.StorageImagesVolume(name) != "" {
```

Comment:
could write this as a for loop iterating on a slice of functions containing config.StorageImagesVolume() and config.StorageBackupsVolume(), to avoid duplication here.